addons/comments-linebreaks/addon.json

```diff
@@ -18,9 +18,43 @@
     {
       "url": "break-spaces.css",
       "matches": ["studios", "projects"]
+    },
+    {
+      "url": "scrollbars.css",
+      "matches": ["studios", "projects"],
+      "if": {
+        "settings": {
+          "scrollbars": true
+        }
+      }
+    }
+  ],
+  "settings": [
+    {
+      "name": "Add scroll bars to tall comments",
+      "id": "scrollbars",
+      "type": "boolean",
+      "default": false
+    }
```

Comment:
Missing comma, that's why workflows are failing

Reply:
I noticed that a couple seconds before you reviewed.